pkg/ui/workspaces/cluster-ui/webpack.config.js

```diff
@@ -115,9 +115,7 @@ module.exports = (env, argv) => {
             {
               loader: "less-loader",
               options: {
-                lessOptions: {
-                  javascriptEnabled: true
-                }
```

Comment:
I imagine this change won't be necessary once we no-longer downgrade less-loader?